lib/liveman_web/router.ex

```diff
@@ -9,15 +10,22 @@ defmodule LivemanWeb.Router do
     plug :put_secure_browser_headers
   end
 
+  pipeline :require_authenticated_user do
+    plug(:authenticate_user)
```

Comment:
Could we remove the parentheses so it matches the rest of the file 😄 
```suggestion
    plug :authenticate_user
```